examples/c++now-allocator.cpp

```diff
@@ -0,0 +1,25 @@
+// examples/c++now-alloctor.cpp                                       -*-C++-*-
```

Comment:
```suggestion
// examples/c++now/allocator.cpp                                       -*-C++-*-
```
Q1: Should we subdirectory instead of repeating this prefix?
Q2: typo? - `alloctor`?